shared/common-adapters/error-profile.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react'
+import Box from './box'
+import Text from './text'
+import {globalColors, globalMargins} from '../styles/style-guide'
+
+export type Props = {
+  error: string
+}
+
+export default function ErrorLoadingProfile ({error}: Props) {
```

Comment:
We should talk to design about this. I think we should likely have a global error component. Like the red bar on top. We could have all smart components have the ability to show that